nbs/40_tabular.core.ipynb

```diff
@@ -1800,7 +1809,10 @@
     "\n",
     "test_series(to.cat_names, ['a', 'b_na'])\n",
     "test_series(to['a'], [1,2,3,2,2,3,1])\n",
-    "test_eq(df.a.dtype,int)\n",
+    "if sys.platform == \"win32\":\n",
```

Comment:
How about:

```
test_eq(df.a.dtype, np.int64 if sys.platform == "win32" else int)
```

Or maybe define a module variable somewhere like

```
default_int = np.int64 if sys.platform == "win32" else int
```

...and then use that?